packages/ui/src/screens/account_details/components/balance/index.tsx

```diff
@@ -26,7 +28,7 @@ type BalanceProps = Parameters<typeof formatBalanceData>[0] & {
 const Balance: FC<BalanceProps> = (props) => {
   const { t } = useTranslation('accounts');
   const { classes, cx, theme } = useStyles();
-  const market = useRecoilValue(readMarket);
+  let market = useRecoilValue(readMarket);
```

Comment:
Why was the reason for changing this from `const` to `let`?

Reply:
Oh actually I was going to change the market so made it `let` but you're right it should be `const.`. I will fix it